pinot-core/src/main/java/com/linkedin/pinot/core/util/AvroUtils.java

```diff
@@ -13,39 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.linkedin.pinot.core.indexsegment.utils;
+package com.linkedin.pinot.core.util;
 
 import com.google.common.base.Preconditions;
 import com.linkedin.pinot.common.data.DimensionFieldSpec;
 import com.linkedin.pinot.common.data.FieldSpec;
 import com.linkedin.pinot.common.data.MetricFieldSpec;
 import com.linkedin.pinot.common.data.Schema;
 import com.linkedin.pinot.common.data.TimeFieldSpec;
-import com.linkedin.pinot.core.data.readers.AvroRecordReader;
+import com.linkedin.pinot.core.data.GenericRow;
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import java.util.zip.GZIPInputStream;
 import org.apache.avro.Schema.Field;
 import org.apache.avro.file.DataFileStream;
+import org.apache.avro.generic.GenericData;
 import org.apache.avro.generic.GenericDatumReader;
 import org.apache.avro.generic.GenericRecord;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 
-/**
- *
- * Aug 19, 2014
- */
 public class AvroUtils {
-  private static final Logger LOGGER = LoggerFactory.getLogger(AvroUtils.class);
+  private AvroUtils() {
+  }
 
   private static final String COUNT = "count";
```

Comment:
I understanding this is outside the scope of this PR, but we should clean this code at some point. Just because a dimension has "count" or "met" substring in the name it treating it as metric is incorrect. This will "country" a metric.

Reply:
Haha, yes, while they are only used in testing.